### PR TITLE
feat(i18n):support nested

### PR DESCRIPTION
**Pull request by 15168440402**
### Features:
- Support nested  [#472](https://github.com/BlazorComponent/MASA.Blazor/pull/472)

### CODE REVIEW

File: src/Doc/Masa.Blazor.Doc/Shared/BaseLayout.razor.cs [hunk 1]
```diff
@@ -89,7 +89,7 @@ protected override void OnInitialized()
 
             ChangeLanguage(lang);
 
-            Navigation.LocationChanged += OnLocationChanged;
+            Navigation.LocationChanged += OnLocationChanged;
```
Comment (doddgu) [inline src/Doc/Masa.Blazor.Doc/Shared/BaseLayout.razor.cs:92]: why add space?

File: src/Doc/Masa.Blazor.Doc/Masa.Blazor.Doc.csproj [hunk 2]
```diff
@@ -22,13 +22,13 @@
...
 	  <Content Update="wwwroot\locale\en-US.json">
-	    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
 	  </Content>
 	  <Content Update="wwwroot\locale\languageConfig.json">
 	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
 	  </Content>
 	  <Content Update="wwwroot\locale\zh-CN.json">
-	    <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+	    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```
Comment (doddgu) [inline src/Doc/Masa.Blazor.Doc/Masa.Blazor.Doc.csproj:31]: Why always?

File: src/Doc/Masa.Blazor.Doc/wwwroot/docs/features/internationalization.en-US.md [hunk 3]
```diff
@@ -115,11 +116,69 @@ void Example()
...
+        "Name":"名称",
+        "Price":"价格"
+    }
+}
+```
+
+- en-US.json
+
+```
+{
```
Comment (doddgu) [inline src/Doc/Masa.Blazor.Doc/wwwroot/docs/features/internationalization.en-US.md:139]: Missing the sample?

File: src/Doc/Masa.Blazor.Doc/wwwroot/docs/features/internationalization.zh-CN.md [hunk 4]
```diff
@@ -115,11 +116,70 @@ void Example()
...
+        "Name":"名称",
+        "Price":"价格"
+    }
+}
+```
+
+- en-US.json
+
+```
+{
```
Comment (doddgu) [inline src/Doc/Masa.Blazor.Doc/wwwroot/docs/features/internationalization.zh-CN.md:140]: Missing the sample

**Review by doddgu — CHANGES_REQUESTED** [review]

File: src/Doc/Masa.Blazor.Doc/Masa.Blazor.Doc.csproj (hunk 2 above)
Comment (15168440402) [reply src/Doc/Masa.Blazor.Doc/Masa.Blazor.Doc.csproj:31]: When debugging the code locally, the json file is often an old file, and the compiler does not copy the file when it is updated in time. It is the safest to choose always copy.

**Review by doddgu — APPROVED** [review]
LGTM

**Pull request merged**